server.go

```diff
@@ -56,7 +57,7 @@ func NewServer(config *Config, backend Backend) Server {
 // afterwards, which is when configuration options are checked.
 func NewServerWithLogger(config *Config, backend Backend, logger *logrus.Logger) Server {
 	router := chi.NewRouter()
-	router.Use(middleware.DefaultCompress)
+	router.Use(middleware.Compress(flate.DefaultCompression))
```

Comment:
DefaultCompress was deprecated tho seems it was accidentally deleted between 4.0.3 and 4.0.4 https://github.com/go-chi/chi/issues/497